test/test_common.py

```diff
@@ -0,0 +1,9 @@
+import pytest
+from app.common import logger
+
+class TestLogger:
+    def setup_method(self):
+        self.logger = logger.Logger()
```

Comment:
not quite sure, this will work... ?

did you tested it ?

should it not be : 
```suggestion
        self.logger = logger()
```
instead ?

Reply:
yes